nailgun/entities.py

```diff
@@ -1771,14 +1771,12 @@ class ContentViewVersion(
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
             'content_view': entity_fields.OneToOneField(ContentView),
-            'environments': entity_fields.OneToManyField(LifecycleEnvironment),
+            'environment': entity_fields.OneToManyField(Environment),
             'major': entity_fields.IntegerField(),
             'minor': entity_fields.IntegerField(),
             'package_count': entity_fields.IntegerField(),
             'puppet_module': entity_fields.OneToManyField(PuppetModule),
             'version': entity_fields.StringField(),
-            'repositories': entity_fields.OneToManyField(Repository),
-            'description': entity_fields.StringField(),
```

Comment:
These fields are correct and work fine with the API. Why remove them? If the convention is to use singular attributes, then let's just change it to 'repository'. Same for 'environment'.

```py
>>> ContentViewVersion(id=3).read()
nailgun.entities.ContentViewVersion(puppet_module=[], major=2, package_count=0, repositories=[nailgun.entities.Repository(id=6)], environments=[nailgun.entities.LifecycleEnvironment(id=2), nailgun.entities.LifecycleEnvironment(id=3)], version=u'2.0', content_view=nailgun.entities.ContentView(id=2), id=3, minor=0, description=u'My Devel content view version!')
```

Reply:
I would actually prefer for the standard to be plural or singular depending on the type of relationship, but I guess changing that will be a lot of effort for codebases like FAM & Robottelo.

But the other benefit of that could be dropping the inflection dependency, which would help towards https://github.com/SatelliteQE/nailgun/issues/493